### ISSUE TITLE
Doubt forum is not responsive 

### ISSUE THREAD

**Issue by Abhijay007** [codeforcauseorg/edu-client#401]
## Describe the bug

The doubt forum is not responsive, the start discussion component is aligned to the extreme right of the page.

## To Reproduce

1. open the website click on the "forum" scroll down 

## Environment (please complete the following information):

- OS: Windows 10 
- Browser: chrome

## Additional context

![doubt forum](https://user-images.githubusercontent.com/64387054/132120350-22ef4e2c-0dbd-4bc6-b7a6-062a06b75d58.PNG)

cc: @adarsh-technocrat @Abhishek-kumar09

**Comment by Abhishek-kumar09** [comment]
Thanks @Abhijay007 Would you like to work on it too?

**Comment by Abhijay007** [comment]
> Thanks @Abhijay007 Would you like to work on it too?

yes

**Comment by Abhishek-kumar09** [comment]
Thanks @Abhijay007

**Comment by yashvardhan05** [comment]
Can i work on this issue?

**Comment by adarsh-technocrat** [comment]
Sure @yashvardhan05  =)

**Comment by yashvardhan05** [comment]
I am not understanding the issue properly, can you please help me with that?

**Comment by adarsh-technocrat** [comment]
Yeah, sure  @yashvardhan05  let me know if you have any doubts !!

**Comment by Abhijay007** [comment]
@yashvardhan05 I can help you to understand the codebase and issue you can ping me here on discord : my discord username and id  :abhijay007#3182 :)

**Comment by yashvardhan05** [comment]
> @yashvardhan05 I can help you to understand the codebase and issue you can ping me here on discord: my discord username and id: abhijay007#3182 :)

Sorry for tuning in late, i've sent you friend request on Discord Btw

**Comment by Kunal0220** [comment]
Hey @Abhishek-kumar09 , has this issue been resolved ? If no , can I work on this issue?

**Comment by yashvardhan05** [comment]
You can work it bro! Sorry for the hold tho

**Comment by yashvardhan05** [comment]
fixed this error need to reply, give final confirmation call on the stand up

**Comment by adarsh-technocrat** [comment]
> fixed this error need to reply, give final confirmation call on the stand up

Hey, @yashvardhan05 this project is no longer active.
